OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/InAppMessageInfo.java

```diff
@@ -0,0 +1,37 @@
+package com.optimove.android.optimobile;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONObject;
+
+import java.util.Date;
+
+/**
+ * Information about an in-app message for interceptor evaluation.
+ */
+public interface InAppMessageInfo {
+    
+    int getMessageId();
+    
+    @Nullable
+    String getPresentedWhen();
```

Comment:
I think we have an enum for this, worth using instead of implementation detail raw string?

Reply:
This class was removed